javascript-light-bulb/main.js

```diff
@@ -0,0 +1,20 @@
+// assign return of querySelector method of document object with 1 argument, 'bulb', to new variable bulb
+// create addEventListener method of bulb object that listens for 'click', and calls new functino click with button parameter
+// assign null to new variable onoff
+// if the button class is 'off'
+// turn it on
+// else (button is already on), turn it off
+
+var container = document.querySelector('.container');
+var bulb = document.querySelector('.bulb');
+bulb.addEventListener('click', function click(button) {
+  var onoff;
+  if (bulb.className === 'bulb off') {
```

Comment:
This is making your JavaScript dependent on the state of the DOM to decide what action to take, and that is bad. You want to be able to know whether to turn the light on or off based on the value of a variable. You should try to think of a way that you can use a variable to keep track of if the light is currently on or off.